packages/passport/sdk/src/zkEvm/zkEvmProvider.ts

```diff
@@ -48,7 +48,7 @@ export class ZkEvmProvider implements Provider {
 
   readonly #guardianClient: GuardianClient;
 
-  readonly #staticJsonRpcProvider: StaticJsonRpcProvider; // Used for read
```

Comment:
Was there a reason for making the variable public? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_properties